apps/backend/src/db/migrations/20250606_init.ts

```diff
@@ -0,0 +1,304 @@
+}
+
+async function createBallotPaperSectionCandidateTable(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .createTable('BallotPaperSectionCandidate')
+    .$call(addDefaultColumns)
+    .addColumn('ballotPaperSectionId', 'uuid', (col) => col.notNull())
+    .addColumn('candidateId', 'uuid', (col) => col.notNull())
+    .addCheckConstraint('modified_after_created', sql`"modifiedAt" >= "createdAt"`)
+    .execute();
+}
+
+async function createCandidateTable(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .createTable('Candidate')
+    .$call(addDefaultColumns)
+    .addColumn('title', 'varchar(256)', (col) => col.notNull())
+    .addColumn('description', 'varchar(256)')
+    .addColumn('electionId', 'uuid', (col) => col.notNull())
+    .addCheckConstraint('modified_after_created', sql`"modifiedAt" >= "createdAt"`)
+    .addCheckConstraint('valid_title', sql`"title" ~ '^[a-zA-Z0-9 .,_:;!?()/\\-]{1,256}$'`)
+    .execute();
+}
+
+async function createVoterGroupTable(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .createTable('VoterGroup')
+    .$call(addDefaultColumns)
+    .addColumn('name', 'varchar(256)', (col) => col.notNull())
+    .addColumn('description', 'varchar(256)')
+    .addColumn('pubKey', 'text')
+    .addColumn('privKey', 'text')
+    .addColumn('voterTokensGenerated', 'boolean', (col) => col.notNull().defaultTo(false))
+    .addCheckConstraint('modified_after_created', sql`"modifiedAt" >= "createdAt"`)
+    .$call(addNameCheckConstraint)
+    .execute();
+}
+
+async function createVoterTable(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .createTable('Voter')
+    .$call(addDefaultColumns)
+    .addColumn('voterGroupId', 'uuid', (col) => col.notNull())
+    .addCheckConstraint('modified_after_created', sql`"modifiedAt" >= "createdAt"`)
+    .execute();
+}
+
+async function createVoterRegisterTable(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .createTable('VoterRegister')
+    .$call(addDefaultColumns)
+    .addColumn('voted', 'boolean', (col) => col.notNull().defaultTo(false))
+    .addColumn('ballotPaperId', 'uuid', (col) => col.notNull())
+    .addColumn('voterId', 'uuid', (col) => col.notNull())
+    .addCheckConstraint('modified_after_created', sql`"modifiedAt" >= "createdAt"`)
+    .execute();
+}
+
+async function createTables(db: Kysely<any>): Promise<void> {
+  await createUserTable(db);
+  await createAccessTokenBlacklistTable(db);
+  await createElectionTable(db);
+  await createBallotPaperTable(db);
+  await createBallotPaperSectionTable(db);
+  await createBallotPaperSectionCandidateTable(db);
+  await createCandidateTable(db);
+  await createVoterGroupTable(db);
+  await createVoterTable(db);
+  await createVoterRegisterTable(db);
+}
+
+// --- Foreign Key Helper Functions ---
+async function addElectionForeignKeys(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .alterTable('Election')
+    .addForeignKeyConstraint(
+      'Election_electionCreatorId_fkey',
```

Comment:
This uses three different string cases :D I would use an enum for this like `ElectionFKs.electionCreatorId = electionElectionCreatorIdFK`. Same thing goes for all other FK names.